### PR TITLE
Torchscript test for Flaubert

### PR DESCRIPTION
**Pull request by LysandreJik**
cc @HamidShojanazeri, test for https://github.com/huggingface/transformers/pull/12292

### CODE REVIEW

File: tests/test_modeling_flaubert.py [hunk 1]
```diff
@@ -401,6 +402,29 @@ def test_model_from_pretrained(self):
...
+            config.torchscript = True
+            model = model_class(config=config)
+
+            inputs_dict = self._prepare_for_class(inputs_dict, model_class)
+            traced_model = torch.jit.trace(
+                model, (inputs_dict["input_ids"].to("cpu"), inputs_dict["attention_mask"].to("cpu"))
+            )
+
+            with tempfile.TemporaryDirectory() as tmp:
+                torch.jit.save(traced_model, os.path.join(tmp, "bert.pt"))
```
Comment (HamidShojanazeri) [inline tests/test_modeling_flaubert.py:424]: @LysandreJik nit: maybe we can change the 'bert.pt' to 'traced_model.pt'.

**Review by HamidShojanazeri — APPROVED** [review]
Thanks @LysandreJik for putting it together, it looks great.

File: tests/test_modeling_flaubert.py [hunk 2]
```diff
@@ -401,6 +402,29 @@ def test_model_from_pretrained(self):
             model = FlaubertModel.from_pretrained(model_name)
             self.assertIsNotNone(model)
 
+    @slow
+    @require_torch_gpu
+    def test_torchscript_device_change(self):
+        config, inputs_dict = self.model_tester.prepare_config_and_inputs_for_common()
+        for model_class in self.all_model_classes:
+
+            # BertForMultipleChoice behaves incorrectly in JIT environments.
```
Comment (LysandreJik) [inline tests/test_modeling_flaubert.py:411]:
```suggestion
            # FlauBertForMultipleChoice behaves incorrectly in JIT environments.
```

File: tests/test_modeling_flaubert.py (hunk 1 above)
Comment (LysandreJik) [inline tests/test_modeling_flaubert.py:424]:
```suggestion
                torch.jit.save(traced_model, os.path.join(tmp, "traced_model.pt"))
```

**Pull request merged**